doc/source/user_guide/fields/field_data.rst

```diff
@@ -95,6 +95,21 @@ To obtain face connectivity data, specify ``FacesConnectivity`` as the ``data_ty
   >>> faces_connectivity_data["inlet"].connectivity[5]
   array([ 4,  5, 12, 11])
 
+You can also query the face connectivity data in raw format:
+
+.. code-block:: python
+
+  >>> faces_connectivity_request = SurfaceFieldDataRequest(
+  >>>     surfaces=[VelocityInlet(settings_source=solver_session, name="inlet")],
+  >>>     data_types=[SurfaceDataType.FacesConnectivity]
+  >>>     raw_data=True,
+  >>> )
+  >>> faces_connectivity_data = field_data.get_field_data(faces_connectivity_request)
+
+  >>> faces_connectivity_data["inlet"].connectivity
```

Comment:
@prmukherj What means do we provide to help users interpret individual faces in the flat raw data array? For example, is there a way to query face-level metadata like face types or offsets?

Reply:
Updated as per discussions.
Thank you.

Reply:
@prmukherj I still feel that it lacks clarity.

Reply:
A full explanation of the format.

Reply:
Done. @seanpearsonuk, could you please have a look if clarity is better?
Thank you,